nsdb-cluster/src/main/scala/io/radicalbit/nsdb/cluster/actor/ClusterListener.scala

```diff
@@ -77,6 +79,12 @@ class ClusterListener() extends Actor with ActorLogging {
     */
   private val nsdbMetrics: mutable.Map[String, Set[NodeMetrics]] = mutable.Map.empty
 
+  /**
+    * Retry policy
+    */
+  private val delay   = 1 seconds
```

Comment:
it's better to set these variables inside the `nsdb.conf` file

Reply:
agree. i think also the parameter below this